Kickstarter-iOS/Views/Cells/ViewMoreRepliesCell.swift

```diff
@@ -0,0 +1,75 @@
+import Library
+import Prelude
+import UIKit
+
+final class ViewMoreRepliesCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var rootStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let titleTextLabel = UILabel(frame: .zero)
+
+  private let viewModel: ViewMoreRepliesCellViewModelType = ViewMoreRepliesCellViewModel()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.bindStyles()
+    self.configureViews()
+    self.bindViewModel()
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Styles
+
+  // TODO: - Internationalize string
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableViewCellStyle()
+
+    _ = self.rootStackView
+      |> commentCellRootStackViewStyle
+      |> \.layoutMargins .~ .init(
+        top: Styles.grid(1),
+        left: Styles.grid(CommentCellStyles.Layout.leftIndentWidth),
+        bottom: Styles.grid(1),
+        right: Styles.grid(1)
+      )
+
+    _ = self.titleTextLabel
+      |> \.font .~ UIFont.ksr_callout()
+      |> UILabel.lens.textColor .~ .ksr_create_700
+      |> UILabel.lens.text .~ localizedString(key: "View more replies", defaultValue: "View more replies")
```

Comment:
Maybe add a  `TODO://` to later add internationalized `Strings` here.

Reply:
Ah you might have missed it a few lines above. It's on line 30 above the method name,